content/aips/STMATIC-MATICX-LDO-SD-EMISSION-ADMIN-UPDATE-AAVE-V3-POLYGON.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Set LDO, stMATIC, MaticX and SD Emission_Admin for Polygon v3 Liquidity Pool
+author: Llamax - Matthew Graham & defijesus
+discussion: https://governance.aave.com/t/arc-ldo-emission-admin-for-polygon-v3-liquidity-pool/10575
```

Comment:
sadly i think aave interface (and perhaps also front matter) cannot handle multiple entries here.

Can you only link one post here and move all the discussions into the document?